MATLAB/environmental_models/env_atmospheric_drag.m

```diff
@@ -0,0 +1,89 @@
+function F_envdrag = env_atmospheric_drag(time,r,v)
+%input: mission time in sec
+% r position in ECI frame in m
+% v velocity in ECI frame in m/s
+% Omega longitude/right ascension of the ascending node in rad
+
+% outputs: Fenv_drag in N in direction of velocity % check dis ECI frame? 
+% assumes a simple, fully static exponentially decaying model (Vallado)
+
+global const
+%time conversions
+UTC_leap = utl_time2datetime(time,const.INITGPS_WN);  %converts mission time in secs to datetime in UTCleapseconds
+%UTC_leap = utl_time2datetime(time,3); test
+offset = 2*86400; %offset of UTCLeapSeconds to UTC time
+del = datetime(0,0,0,0,0,offset);
+del.TimeZone = 'UTCLeapSeconds';
+UTC = UTC_leap - offset;
+
+% r: satellite position in ECI
+%convert ECI coordinates to latitude, longitude, altitude (LLA) geodetic coordinates based on the Universal Coordinated Time (UTC) you specify
+lla = eci2lla(r, [UTC.Year UTC.Month UTC.Day UTC.Hour UTC.Minute UTC.Second]); 
+
+h = abs(lla(3)); %altitude of Position (m); geocentric altitude = geodetic altitude
+
+rho = get_rho(h);
+
+Cd = 0.8; %drag coefficient of angled cube
+A = 0.1*sqrt(2)*0.3; %largest planar area of satellite in m^2
+v_rel = v - cross(const.earth_rate_ecef,r); %velocity relative to the rotating atmosphere
+%v_rel = v - cross([0;0;7.2921158553E-5],r); test
+
+F_envdrag = -0.5*rho*Cd*A*(v_rel*v_rel'); %drag calculated in eci frame
```

Comment:
`v_rel` is a column vector as well, so i think this should be `v_rel'*v_rel`.

Reply:
It looks like F_envdrag is a scalar. Should it be a vector?
F_envdrag = -0.5*rho*Cd*A*(v_rel*v_rel')*(v/norm(v));

Reply:
F_envdrag = -0.5rhoCdA(v_relv_rel')(v_rel/norm(v_rel));
Actually use v_rel instead of v for the vector direction

Reply:
updated